flow360/component/simulation/framework/base_model.py

```diff
@@ -546,14 +546,14 @@ def _convert_dimensions_to_solver(
         assert mesh_unit is not None
 
         for property_name, value in self_dict.items():
-            if property_name in [COMMENTS, TYPE_TAG_STR] + exclude:
+            if property_name in [COMMENTS, TYPE_TAG_STR]:
```

Comment:
Can you add a comment/docstring in the function saying what this exclude actually means so other developers know?